accounts/views.py

```diff
@@ -69,13 +72,14 @@ def signup(request):
         ends_time = timezone.now() + timedelta(days=14)
 
         subscription = Subscription.objects.get_or_create(
-            plan_id=1,
+            plan=plan,
```

Comment:
When the new user signup on the system, the free subscription should be create automatic for that user

and free subscription, is linked by plan created by system admin